client/state/themes/schema.js

```diff
@@ -112,3 +112,7 @@ export const themeFiltersSchema = {
 	},
 	additionalProperties: false,
 };
+
+export const themesBannerVisibleSchema = {
+	type: [ 'boolean', 'null' ],
```

Comment:
We shouldn't (need to) allow for `null` :slightly_smiling_face:

Reply:
(For the same reason as https://github.com/Automattic/wp-calypso/pull/24335/files#r183109665)

Reply:
Let's change this to `type: 'boolean'`